### PR TITLE
Verify Spelling

### PR DESCRIPTION
**Pull request by gambol99**
- adding a verifiy-spelling into the ci checks

### CODE REVIEW

File: docs/apireference/build/node_modules/jquery.scrollto/CHANGELOG.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 ### Compat
 - Dropped support for $.scrollTo.window() and $(window)._scrollable()
 ### Fix
-- Now works consistenly on Chrome 40
+- Now works consistently on Chrome 40
```
Comment (justinsb) [inline docs/apireference/build/node_modules/jquery.scrollto/CHANGELOG.md]: Can we add directories to exclude? This is a library, I believe

File: docs/releases/1.9-NOTES.md [hunk 2]
```diff
@@ -309,7 +309,7 @@ None known at this time
 * Typo fix "etcd manager"->"etcd-manager" [@AdamDang](https://github.com/AdamDang) [#4628](https://github.com/kubernetes/kops/pull/4628)
 * Typo fix "provides"->"provide" [@AdamDang](https://github.com/AdamDang) [#4622](https://github.com/kubernetes/kops/pull/4622)
 * Typo instance group ->"InstanceGroup" [@AdamDang](https://github.com/AdamDang) [#4621](https://github.com/kubernetes/kops/pull/4621)
-* Typo "offical"->"official" [@AdamDang](https://github.com/AdamDang) [#4637](https://github.com/kubernetes/kops/pull/4637)
+* Typo "official"->"official" [@AdamDang](https://github.com/AdamDang) [#4637](https://github.com/kubernetes/kops/pull/4637)
```
Comment (justinsb) [inline docs/releases/1.9-NOTES.md]: Well this is a bit of a problem... I think we can probably live with it here, but is there any way to skip certain lines?

File: pkg/openapi/openapi_generated.go [hunk 3]
```diff
@@ -25960,7 +25960,7 @@ func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenA
 						},
 						"enableTLSAuth": {
 							SchemaProps: spec.SchemaProps{
-								Description: "EnableTLSAuth indicats client and peer TLS auth should be enforced",
+								Description: "EnableTLSAuth indicators client and peer TLS auth should be enforced",
```
Comment (justinsb) [inline pkg/openapi/openapi_generated.go]: Probably `indicates`

**Comment by justinsb** [comment]
Very cool.  Is there a way to exclude a directory (in particular spell checking those jquery libraries!)

#5085 could help us a bunch though - we might be able to get our docs into a separate branch / repo

File: docs/apireference/build/node_modules/jquery.scrollto/CHANGELOG.md (hunk 1 above)
Comment (gambol99) [reply docs/apireference/build/node_modules/jquery.scrollto/CHANGELOG.md]: nothing native in the command line .. but we are just piping the output of [find and excluding with sed](https://github.com/kubernetes/kops/pull/5277/files#diff-b67911656ef5d18c4ae36cb6741b7965R507) .. I'll add node_modules and revert the change

File: docs/releases/1.9-NOTES.md (hunk 2 above)
Comment (gambol99) [reply docs/releases/1.9-NOTES.md]: yeah .. I'd like to keep the releases directory in there .. call this a one off :-)

File: pkg/openapi/openapi_generated.go (hunk 3 above)
Comment (gambol99) [reply pkg/openapi/openapi_generated.go]: I'd fixed in the source but needed to regenerate the openapi

**Comment by justinsb** [comment]
/approve
/lgtm

Thanks for the tweaks; this will really help us out as well!

**Pull request merged**